cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -68,9 +68,9 @@ macro(swift_common_standalone_build_config_llvm product is_cross_compiling)
     set(${product}_NATIVE_LLVM_TOOLS_PATH "${LLVM_TOOLS_BINARY_DIR}")
   endif()
 
-  find_program(SWIFT_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
+  find_program(LLVM_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
```

Comment:
This code is only executed by the out-of-tree build. In theory we don't need to set this at all, but I wanted to minimize changes in behavior to just the changes I needed to make this work.

The motivator for this is making the in-tree cross-compile work correctly, so that is where my testing has been focused.